tests/src/test/scala/system/basic/WskRuleTests.scala

```diff
@@ -91,7 +92,7 @@ abstract class WskRuleTests extends TestHelpers with WskTestHelpers {
     val active = Some("active".toJson)
     val inactive = Some("inactive".toJson)
     val statusPermutations =
-      Seq((triggerName, active), (triggerName, inactive), (triggerName2, active), (triggerName, inactive))
+      Seq((triggerName, active), (triggerName, inactive), (triggerName2, active), (triggerName2, inactive))
```

Comment:
I believe this was a typo that I have fixed here?

Reply:
Nope needs to be `triggerName` otherwise it's just running the first two permutations again.